Makefile

```diff
@@ -48,6 +48,7 @@ install: bin bin/$(BINARY_NAME)_ui
 # Clean up images and binaries
 clean:
 	rm -f bin/wego pkg/flux/bin/flux
+	rm -f pkg/flux/bin/fluxchecksums.txt
```

Comment:
nit - have the first line remove bin/wego and this line remove everything under pkg/flux - unless we want to keep files in that dir other than flux and fluxchecksums.txt

Reply:
there are no other files